synapse/push/clientformat.py

```diff
@@ -71,24 +74,24 @@ def _add_empty_priority_class_arrays(d: Dict[str, list]) -> Dict[str, list]:
     return d
 
 
-def _rule_to_template(rule: Dict[str, Any]) -> Optional[Dict[str, Any]]:
-    unscoped_rule_id = None
-    if "rule_id" in rule:
-        unscoped_rule_id = _rule_id_from_namespaced(rule["rule_id"])
```

Comment:
Did everything always previously have a rule ID anyway?

Reply:
I.e. was this if check useless?

Reply:
Yeah, the DB has a not null check for `rule_id` for example.